server/database.py

```diff
@@ -22,13 +23,14 @@ def get_id(self):
 
 
 class User(AsceeModel, UserMixin):
-    is_admin = props.Bool(default=False)
-    is_recruiter = props.Bool(default=False)
-    is_senior_recruiter = props.Bool(default=False)
+    is_admin = props.Bool(indexed=True, default=False)
+    is_recruiter = props.Bool(indexed=True, default=False)
+    is_senior_recruiter = props.Bool(indexed=True, default=False)
     recruiter_id = props.Integer(indexed=True, optional=True)
     status_level = props.Integer(indexed=True, default=0)
+    name = props.String(default="Unknown")
 
-    STATUS_LIST = ('new', 'escalated', 'accepted', 'rejected')
+    STATUS_LIST = ('new', 'claimed', 'escalated', 'accepted', 'rejected')
```

Comment:
'claimed' shouldn't be a status because this duplicates information. If `recruiter_id` is `None`, the applicant is unclaimed. Please remove this status.

We could add in a helper function for determining if a character is unclaimed, if you like, or you can use `char.recruiter_id == None` or `char.recruiter_id != None`.

Reply:
we can, but it complicated the front end that has logic based on these status. On the database keep it how you like, but the API should just pass the status